huaweicloud/services/aom/resource_huaweicloud_aom_uniagent_batch_install.go

```diff
@@ -0,0 +1,268 @@
+package aom
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var aomUniAgentBatchInstallNonUpdatableParams = []string{"agent_import_param_list", "proxy_region_id",
```

Comment:
```suggestion
var uniAgentBatchInstallNonUpdatableParams = []string{"agent_import_param_list", "proxy_region_id",
```
do not repeat the package name again

Reply:
done